core/src/main/java/io/fabric8/maven/core/handler/ProbeHandler.java

```diff
@@ -46,6 +47,9 @@ public Probe getProbe(ProbeConfig probeConfig)  {
             probe.setTimeoutSeconds(timeoutSeconds);
         }
         HTTPGetAction getAction = getHTTPGetAction(probeConfig.getGetUrl());
+        if(StringUtils.isEmpty(getAction.getHost())) {
```

Comment:
This check can lead to a NPE since getAction can be null. Could explain briefly why an empty String as host is an issue ?